### PR TITLE
chore: Change ci_autosave_final_commit message

### PR DESCRIPTION
**Pull request by realodix**

### CODE REVIEW

File: .PyFunceble.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ aggressive: false
 auto_continue: true
 ci: true
 ci_autosave_commit: '[Autosave][Dead-Hosts::repository-structure]'
-ci_autosave_final_commit: '[Final/Result][Dead-Hosts::repository-structure]'
+ci_autosave_final_commit: '[Final/Result] Dead-Hosts: AdBlockID Plus'
```
Comment (funilrys) [inline .PyFunceble.yaml:6]: Is something not working @realodix ? Should I look further into it ?

**Comment by realodix** [comment]
How change the commit message? https://github.com/dead-hosts/AdBlockID_git_realodix/pull/4 it looks not working.

File: .PyFunceble.yaml (hunk 1 above)
Comment (realodix) [reply .PyFunceble.yaml:6]: How change the commit message? https://github.com/dead-hosts/AdBlockID_git_realodix/pull/4 it looks not working.

Comment (funilrys) [reply .PyFunceble.yaml:6]: Apparently it is working: https://github.com/dead-hosts/AdBlockID_git_realodix/blob/9b9646ca3e7991ad62f66e77e0715f2bef1cc9bd/info.json#L5

**Pull request merged**